content/applications/sales/sales/amazon_connector/setup.rst

```diff
@@ -2,71 +2,29 @@
-in the page that displays, click on the ``Authorize new developper`` button:
-
-.. image:: ./media/seller_central_apps.png
-  :align: center
-
-Fill in the form depending on your Marketplace:
-
-.. image:: ./media/seller_central_form.png
-  :align: center
-
-If your seller account is registered in the **North America** region, use these values:
-
-- Developer's Name: ``Odoo S.A.``
-- Developer ID: ``586127723692``
-
-For the **Europe** region, use these values:
-
-- Developer's Name: ``Odoo S.A.``
-- Developer ID: ``579095187166``
-
-Amazon will then inform you that by submitting the form, you are giving access
-to your Seller Central Information to Odoo S.A.
-
-.. note::
-    Odoo S.A. is unable to access your Amazon account's information without the
-    Authorization Token which is stored in your Odoo database - we do not store
-    these tokens on our platform directly and are therefore unable to access
-    your account's information outside of the Amazon Connector normal flows.
-
 Register your Amazon account in Odoo
 ====================================
 
 .. _amazon/setup:
 
 To register your seller account in Odoo, navigate to :menuselection:`Sales --> Configuration
---> Settings --> Connectors --> Amazon Sync --> Amazon Accounts` and click on **CREATE**.
+--> Settings --> Connectors --> Amazon Sync --> Amazon Accounts` and click on :guilabel:`CREATE`.
 
-The **Seller ID** can be found in Seller Central under the link **Your Merchant Token** on the
-**Seller Account Information** page. The **Authorization Token** is the one you generated in the
-:ref:`previous step <amazon/generate_auth_token>`.
+#. Choose a meaningful name for your account (e.g. `Europe`), and select your :guilabel:`Sign-up
+   Marketplace`. This is the original sign-up marketplace of your seller account (e.g. if you
+   created your account on Amazon Germany, your sign-up marketplace will be `amazon.de`)
 
-Upon saving, your credentials are checked. In case of issues, an error will be displayed - the
-information cannot be saved until your credentials are recognized by Amazon.
+#. Upon saving, a button to :guilabel:`link with Amazon` appears. Click on it to be redirected to
+   the Amazon login page, or directly to the consent page if you are already logged in. There,
+   confirm that you want Amazon to give Odoo access to your account and related data.
+
+#. Amazon redirects you to Odoo, with your account registered.
 
 Once the account is registered, the marketplaces available to this account are synchronized and
-listed under the **Marketplaces** tab. If you wish, you can remove some items from the list of
-synchronized marketplaces to disable their synchronization.
+listed under the :guilabel:`Marketplaces` tab. If you wish, you can remove some items from the list
+of synchronized marketplaces to disable their synchronization.
+
+In case of issues, click on :guilabel:`unlink Account` and link it again anytime you want to.
```

Comment:
I wouldn't advertise that (or at least not in the "register your account section), especially since it's only available in debug mode.

Reply:
I delete it, I feel it doesn't have it's place in the `manage` file either. But I can add it there if you wish =]